pages/index.vue

```diff
@@ -29,4 +35,15 @@ const { data: packageInfo } = await useAsyncData<PackageInfo>(() =>
 );
 
 const qiskitVersion = packageInfo.value?.info.version ?? "";
+
+useSchemaOrg([
+  defineWebSite({
+    name: "Qiskit.org",
+  }),
+  defineOrganization({
+    name: "Qiskit",
+    logo: "public/images/qiskit-logo.png",
```

Comment:
@memolina2323 - can you please double-check if we need the `public/` part of this file path? 

It seems like with this, it'll render to:

`https://qiskit-org-pr-3243.dcq4xc5i083.us-south.codeengine.appdomain.cloud/public/images/qiskit-logo.png`

I used [this Schema Markup Validator](https://validator.schema.org/#url=https%3A%2F%2Fqiskit-org-pr-3243.dcq4xc5i083.us-south.codeengine.appdomain.cloud%2F), to test the Branch preview URL, but let me know if I'm doing that incorrectly.

Thanks!

Reply:
I think you're right, the public part is not needed, thanks for pointing that out!